Presentation.Web/app/components/it-advice/it-advice-modal-dialog.ts

```diff
@@ -15,19 +15,19 @@
                     type,
                     action,
                     object,
-                    currentUser: Kitos.Services.IUser,
-                    entityMapper: Kitos.Services.LocalOptions.IEntityMapper,
+                    currentUser: Services.IUser,
+                    entityMapper: Services.LocalOptions.IEntityMapper,
                     adviceData) => {
                     $scope.hasWriteAccess = hasWriteAccess;
                     $scope.selectedReceivers = [];
                     $scope.selectedCCs = [];
                     $scope.adviceTypeData = null;
                     $scope.adviceRepetitionData = null;
-                    $scope.adviceTypeOptions = Kitos.Models.ViewModel.Advice.AdviceTypeOptions.options;
-                    $scope.adviceRepetitionOptions = Kitos.Models.ViewModel.Advice.AdviceRepetitionOptions.options;
+                    $scope.adviceTypeOptions = Models.ViewModel.Advice.AdviceTypeOptions.options;
+                    $scope.adviceRepetitionOptions = Models.ViewModel.Advice.AdviceRepetitionOptions.options;
 
-                    $scope.multipleEmailValidationRegex =
-                        "(([a-zA-Z\\-0-9\\.]+@)([a-zA-Z\\-0-9\\.]+)\\.([a-zA-Z\\-0-9\\.]+)(, )*)+";
+                    //Format {email1},{email2}. Space between , and {email2} is ok but not required
+                    $scope.multipleEmailValidationRegex = "(([a-zA-Z\\-0-9\\.]+@)([a-zA-Z\\-0-9\\.]+)\\.([a-zA-Z\\-0-9\\.]+)((,)( )*)*)+";
```

Comment:
Hvordan håndteres "test@test.dk, , , , tester@test.dk"? Den regex ser ud til at tillade mange kommaer uden gyldige emails imellem

Reply:
(([a-zA-Z\-0-9\.]+@)([a-zA-Z\-0-9\.]+)\.([a-zA-Z\-0-9\.]+)((,)*( )*){1})+

Denne virker muligvis bedre? (Mangler lige at blive formateret til visual studio med escapes)

Reply:
mja lige det problem er endnu ikke løst med fixet. Jeg ser lige om din løsning virker

Reply:
Den var heller ikke helt rigtig for den KRÆVEDE en (,)( )* konstallation uanset om der kom en email nr 2 eller ej. Det er der heldigvis råd for ;-)

![image](https://user-images.githubusercontent.com/13116889/119438636-f57ac900-bd20-11eb-9bfe-5e0541ae06d0.png)

Reply:
Og som om det ikke var nok så manglede vi også situationen mm@mm.dk@aa@aa.dk som jo er ugyldig men det tillod den,,, så nu:

![image](https://user-images.githubusercontent.com/13116889/119439743-47245300-bd23-11eb-8c2d-55f9287424e1.png)

Her er den rekursiv i den forstand at den benævner at hvis der er noget i feltet SKAL der være mindst 1 email og så må der være 0..* blokker af ((,)( )*{email}) blokke. Dermed har vi håndteret de mønstre der skal til